proposals/0293-vehicle-type-filter.md

```diff
@@ -388,30 +458,26 @@ The below example shows valid vehicle type filters.
 ```
 #### Determining Vehicle Type Info
 
-1. After version negotiations and starting the RPC service, the proxy should check if `GetVehicleType` is supported by the SDL enabled IVI system.
-2. If the protocol version supports `GetVehicleType`, the proxy will send the `GetVehicleType` protocol message.
-3. On receiving vehicle type information, proxy will check `SDLSupportedVehicleTypes` keys to check supported vehicle types.
-4. If the vehicle type is supported, the proxy should also notify the app about the connected vehicle type so that the app can configure SDL as required. The iOS proxy can implement an `OnSDLEnabled` notification similar to that of the Android proxy to notify the app about SDL connection with the supported vehicle. The app can provide lifecycle configuration to the proxy upon receiving the `OnSDLEnabled` notification. 
-5. If the proxy determines from `GetVehicleType` ACK that the vehicle type is not supported, the proxy will end the RPC session. The application layer will not be notified about the vehicle type.
-6. If the protocol version does not support the `GetVehicleType` protocol message, the SDL proxy will continue with the app registration and it will rely on vehicle type information received in the `RegisterAppInterface` response. If vehicle type is not supported, the exclusive apps will be allowed to unregister from the SDL enabled system.
-7. If the `GetVehicleType` message response is NACK, the proxy will end the RPC session and the app will not be registered on the SDL system.
-
-![Sequence Diagram](../assets/proposals/0293-vehicle-type-filter/ios.png)
+1. The proxy will receive vehicle type info in `StartServiceAck` protocol message. 
+2. On receiving vehicle type information, proxy will check `SDLSupportedVehicleTypes` keys to check supported vehicle types.
+3. If the vehicle type is supported, the proxy should also notify the app about the connected vehicle type so that the app can configure SDL as required. 
+4. If the proxy determines that connected vehicle type is not supported, the proxy will end the RPC session. The application will not register on the SDL enabled IVI system.
+5. If the vehicle type information is not available in `StartServiceAck` protocol message or the protocol version is less than that supporting the vehicle type info, the SDL proxy will continue with the app registration and it will rely on vehicle type information received in the `RegisterAppInterface` response. If vehicle type is not supported, the exclusive apps will be allowed to unregister from the SDL enabled system.
```

Comment:
```suggestion
5. If the vehicle type information is not available in `StartServiceAck` protocol message or the protocol version is less than that supporting the vehicle type info, the SDL app library will continue with the app registration and it will rely on vehicle type information received in the `RegisterAppInterface` response. If vehicle type is not supported, the exclusive apps will be allowed to unregister from the SDL enabled system.
```